package.json

```
@@ -77,7 +77,7 @@
     ]
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.1.0",
+    "@docusaurus/module-type-aliases": "2.2.0",
     "@docusaurus/plugin-google-gtag": "^2.2.0",
```

```suggestion
    "@docusaurus/plugin-google-gtag": "2.2.0",
```

Let's take the opportunity to fix this too (and update the lock file as well).